front/components/assistant/conversation/input_bar/editor/MentionList.tsx

```diff
@@ -101,11 +103,15 @@ export const MentionList = forwardRef(function MentionList(
   return (
     <div
       className={classNames(
-        "flex flex-col gap-y-1 overflow-y-auto rounded-xl border px-3 py-2 shadow-xl",
+        "flex min-w-72 flex-col gap-y-1 overflow-y-auto rounded-xl border px-3 py-2 shadow-xl",
         "border-border bg-background dark:border-border-night dark:bg-primary-950"
       )}
     >
-      {props.items.length ? (
+      {props.isLoading ? (
+        <div className="flex justify-center py-2">
+          <Spinner variant="dark" size="sm" />
+        </div>
+      ) : props.items.length ? (
         props.items.map((item, index) => (
```

Comment:
Initially wanted to use the `DropdownMenu` component. It turns out to be really tricky to mount it to the right location.. I will add a card to tackle it later